core/src/main/java/de/gurkenlabs/litiengine/environment/tilemap/MapUtilities.java

```diff
@@ -63,18 +63,26 @@ public static Rectangle2D getTileBoundingBox(final IMap map, final Rectangle2D b
     final int maxY = (int) MathUtilities.clamp(box.getMaxY(), 0, map.getSizeInPixels().height - 1);
     final Point minTilePoint = map.getOrientation().getTile(minX, minY, map);
     final Point maxTilePoint = map.getOrientation().getTile(maxX, maxY, map);
-    int minTileX = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? minTilePoint.x : MathUtilities.clamp(minTilePoint.x, 0, map.getWidth() - 1);
-    int minTileY = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? minTilePoint.y : MathUtilities.clamp(minTilePoint.y, 0, map.getHeight() - 1);
-    int maxTileX = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? maxTilePoint.x : MathUtilities.clamp(maxTilePoint.x, 0, map.getWidth() - 1);
-    int maxTileY = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? maxTilePoint.y : MathUtilities.clamp(maxTilePoint.y, 0, map.getWidth() - 1);
-    final Rectangle2D minTileBounds = map.getOrientation().getBounds(new Point(MathUtilities.clamp(minTileX, 0, map.getWidth() - 1), MathUtilities.clamp(minTileY, 0, map.getHeight() - 1)), map);
-    final Rectangle2D maxTileBounds = map.getOrientation().getBounds(new Point(MathUtilities.clamp(maxTileX, 0, map.getWidth() - 1), MathUtilities.clamp(maxTileY, 0, map.getHeight() - 1)), map);
-
-    return new Rectangle2D.Double(minTileBounds.getX(), minTileBounds.getY(), maxTileBounds.getMaxX() - minTileBounds.getX(), maxTileBounds.getMaxY() - minTileBounds.getY());
+    int minTileX = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? minTilePoint.x
+        : MathUtilities.clamp(minTilePoint.x, 0, map.getWidth() - 1);
+    int minTileY = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? minTilePoint.y
+        : MathUtilities.clamp(minTilePoint.y, 0, map.getHeight() - 1);
+    int maxTileX = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? maxTilePoint.x
+        : MathUtilities.clamp(maxTilePoint.x, 0, map.getWidth() - 1);
+    int maxTileY = map.getOrientation().getName().equals(MapOrientations.ISOMETRIC.getName()) ? maxTilePoint.y
+        : MathUtilities.clamp(maxTilePoint.y, 0, map.getWidth() - 1);
+    final Rectangle2D minTileBounds = map.getOrientation()
+        .getBounds(new Point(MathUtilities.clamp(minTileX, 0, map.getWidth() - 1), MathUtilities.clamp(minTileY, 0, map.getHeight() - 1)), map);
+    final Rectangle2D maxTileBounds = map.getOrientation()
+        .getBounds(new Point(MathUtilities.clamp(maxTileX, 0, map.getWidth() - 1), MathUtilities.clamp(maxTileY, 0, map.getHeight() - 1)), map);
+
+    return new Rectangle2D.Double(minTileBounds.getX(), minTileBounds.getY(), maxTileBounds.getMaxX() - minTileBounds.getX(),
+        maxTileBounds.getMaxY() - minTileBounds.getY());
```

Comment:
The line breaks here are inconsistent. I think manual adjustment of the line break is probably best here.